### ISSUE TITLE
CMD+R doesn't work on Mac

### ISSUE THREAD

**Issue by v-xuanzh** [Microsoft/AzureStorageExplorer#1332]
**Storage Explorer Version:** 1.8.0_20190423.4
**Platform/OS:** macOS High Sierra
**Architecture:** ia32/x64
**Commit:** 2b9f35e1
**Regression From:** Not a regression

**Steps to reproduce:**
1. Make sure AzCopy is enabled.
2. Open one blob container -> Try to upload one blob to it.
3. After uploading successfully, Press CMD+R to refresh the list(Ignore the prompting to refresh).
4. Check the result.

**Expect Experience:**
The file lists can be refreshed.

**Actual Experience:**
Nothing happens.

**More info:**
This issue doesn't reproduce on Windows/Linux.

**Comment by craxal** [comment]
Does the explorer actually have focus?

**Comment by v-xuanzh** [comment]
Yes. The Explorer have focuse.

**Comment by v-xuanzh** [comment]
This issue has been fixed on latest release candidate build. Could you help to resolve it?
